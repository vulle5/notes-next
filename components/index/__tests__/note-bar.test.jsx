import React from 'react'
import { render } from '@testing-library/react'

import NoteBar from '../note-bar'

test('should render a note-bar component', () => {
  const { container } = render(<NoteBar />)
  expect(container.firstChild).toMatchSnapshot()
})
