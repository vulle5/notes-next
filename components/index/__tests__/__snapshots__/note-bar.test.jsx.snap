// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a note-bar component 1`] = `
<div
  class="jsx-4019227191 container"
>
  <div>
    <form
      class="jsx-3726941156 wrapper"
    >
      <textarea
        class="jsx-1612382694 "
        id="note-content"
        name="note-content"
        placeholder="Remind me to..."
        rows="1"
        type="text"
      />
    </form>
  </div>
</div>
`;
